### PR TITLE
Edwin navbar

### PR DESCRIPTION
**Pull request by egichuhis**
Hello there, kindly review my PR and advice.

### Changes made
- Add a README file
- Add MIT Licence
- Add markup index.html for the homepage
- Add style.css stylesheet
- Add font awesome to the project

### CODE REVIEW

File: License [hunk 1]
```diff
@@ -0,0 +1,21 @@
+MIT License
+
+Copyright (c) [year] [fullname]
```
Comment (egichuhis) [inline License:3]: I think you should include your full name and year in this part instead of the default place holders

File: index.html [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+    <body>
+        <header>
+            <nav>
+                <ul class="menu">
+                    <li><i class="fa-solid fa-house"></i>Home</li>
+                    <li><i class="fa-solid fa-id-card-clip"></i>Shorts</li>
+                    <li><i class="fa-solid fa-hand-pointer"></i>Subscriptions</li>
+                    <li><i class="fa-solid fa-bookmark"></i>Library</li>
+                    <li><i class="fa-solid fa-clock-rotate-left"></i>History</li>
+                </ul>
```
Comment (egichuhis) [inline index.html:22]: I think your menu will look better if you added the class "menu" and gave it "list-style: none"

.menu { 
    list-style: none;
    }

**Review by egichuhis — COMMENTED** [review]
Your nav menu

image


Desired nav menu with list style:none;

image

**Review by BregornOriginal — APPROVED** [review]
## Hi @egichuhis 🟢 🟢 🟢 

## Your project is complete!  There is nothing else to say other than... it's time to merge it 🥇 

##Congratulations! 💯 💯 💯 

## Highlights

✅ Linters are passing
✅ Good commit history
✅ Greate review, using friendly comments to make changes
✅ Good job catching bugs! 🐛 

giphy (1)


## Optional suggestions
Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application.
Cheers and Happy coding!:aplauso::aplauso::aplauso:
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

Please, remember to tag me in your question so I can receive the notification.
------
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)[.](https://airtable.com/shrQAqnBwek5a0O0s)._)

**Review by egichuhis — COMMENTED** [review]
Good job!
Excellent commit messages!
Awesome GitHub flow!
Just work on the few changes and cheers!

soclosegif

**Review by amejid — COMMENTED** [review]
Hi @egichuhis, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
- Good use of Github features :heavy_check_mark: 
- Comments are polite :heavy_check_mark: 
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**